### PR TITLE
Prepare for v0.11 release

### PR DESCRIPTION
**Pull request by gandro**
See https://github.com/cilium/hubble/blob/master/RELEASE.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,105 @@ All notable changes to this project will be documented in this file.
...
+* build(deps): bump golang.org/x/sys from 0.3.0 to 0.4.0 (#835, @dependabot[bot])
+* build(deps): bump golangci/golangci-lint-action from 3.2.0 to 3.3.0 (#807, @dependabot[bot])
+* build(deps): bump golangci/golangci-lint-action from 3.3.0 to 3.3.1 (#818, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.47.0 to 1.48.0 (#763, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.48.0 to 1.49.0 (#784, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.49.0 to 1.50.0 (#797, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.50.0 to 1.50.1 (#800, @dependabot[bot])
+* build(deps): bump google.golang.org/grpc from 1.50.1 to 1.51.0 (#819, @dependabot[bot])
+* build(deps): bump google.golang.org/protobuf from 1.28.0 to 1.28.1 (#771, @dependabot[bot])
+* CHANGELOG.md: fix PR ref in the v0.10.0 release note (#753, @kaworu)
```
Comment (kaworu) [inline CHANGELOG.md:85]: nit: IMHO all the dependabot entries should be removed besides maybe Go version bump

**Review by kaworu — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,105 @@ All notable changes to this project will be documented in this file.
...
+Cilium v1.13: Hubble now has visibility into Cilium's SockLB,
+meaning it is possible to observe service address translations performed
+by Cilium on the socket level (#816). Hubble CLI v0.11 also supports the newly
+introduced Cilium v1.13 flow filters for workload and trace ID (#794, #795).
+
+This release also removes the old and deprecated JSON formatter and now always
+uses to the more flexible proto3-based `jsonpb` output when JSON is selected
+as the output format. This is a potentially breaking change and requires that
+e.g. `jq` queries of the form `hubble observe -o json | jq .source` are
+rewritten as `hubble observe -o json | jq .flow.source` (#826).
```
Comment (rolinh) [inline CHANGELOG.md:20]: Could we also list #826 under a `**Breaking Changes**` section below?

File: CHANGELOG.md (hunk 1 above)
Comment (gandro) [reply CHANGELOG.md:85]: So far, we've kept them - as it is generated right now, each commit on the branch has a matching entry here. The script warns if there are commits without a PR for them.

I agree that this is noisy and probably should be collapsed - but I'd prefer it the release script would do that by itself: https://github.com/cilium/release/

File: CHANGELOG.md (hunk 2 above)
Comment (gandro) [reply CHANGELOG.md]: Sure! I've updated the PR to mirror the breaking changes sections we had in previous releases.

**Review by rolinh — APPROVED** [review]

**Pull request merged**